src/utils/chains.ts

```diff
@@ -62,95 +77,65 @@ export function useCurrentChainFullName() {
 }
 
 interface ChainEntry {
+    name: string
     config: Chain
     configExtension: ChainConfigExtension
-    symbolicName: string
 }
 
-const chainEntriesByIdOrName: Partial<Record<string | number, ChainEntry | null>> = {}
-
-function getChainEntry(chainIdOrName: string | number) {
```

Comment:
This one got simplified nicely in `master` because it does now take a strict `ChainKey` value as the sole argument (no second-guessing).